p2p/libp2p/netMessenger.go

```diff
@@ -59,8 +59,6 @@ const (
 	withoutMessageSigning messageSigningConfig = false
 )
 
-var log = logger.GetOrCreate("p2p/libp2p")
```

Comment:
we'll talk about how to completely remove the "github.com/multiversx/mx-chain-logger-go" reference from this file :)